### ISSUE TITLE
[BUG_SP]Convoy mode started by Autodrive 

### ISSUE THREAD

**Issue by mdradek** [Courseplay/Courseplay_FS22#783]
**Steps to reproduce**
Setup course for two or more machines, start it by Autodrive mode "pickup and deliver"

**Actual behavior**
After Autodrive reaching field CP turns on but won't starting work. I noticed there's condtition under which CP start work in convoy - course need to be started at both machines. Therefore second machine driven by Autodrive is unable to reach field point and CP isn't able to takeover work.

**Expected behavior**
CP should start course independently, not first machine waits for second one. 

**Screenshots**
Link to the video clearly showing the issue:

https://drive.google.com/file/d/1oBKH2mbpP5XPrSePH-fGtZjsbqI2kSs_/view?usp=sharing

**Courseplay version**
Version used: 7.0.0.24
Autodrive version: 2.0.0.4 with CP cooperation added

**Map**
Elmcreek

**Comment by Tensuko** [comment]
So the CP Driver is in the way for the 2nd one, right ?
I haven't check how AD works now with CP and how to set the offset lanes.

I guess this is more an Enhancement.

**Comment by mdradek** [comment]
Exactly, if first driver would start work, second one could reach point and let CP to takeover.

When using single tool, everything working perfectly fine.

**Comment by regs1980** [comment]
Ich schildere das Problem jetzt mal in Deutsch, Fahrer 1 beginnt nicht mit der Feldarbeit bevor nicht Fahrer 2 und 3 durch AD auch aktiviert werden, dieses kann aber nicht passieren weil Fahrer 1 nicht AD Zielpunkt verlässt, weil er drauf wartet das Fahrer 2 und 3 auch mit dem Konvoi Starten.
  
<img width="1383" alt="2022-01-31 06_49_31-Farming Simulator 22" src="https://user-images.githubusercontent.com/13314443/151746465-eb4b45e9-87f6-4403-9075-0f49243144c8.png">

**Comment by Tensuko** [comment]
I just wanted to confirm this, but yea okay got it :) I have an Idea how to solve this, but we need to see.

**Comment by Tensuko** [comment]
Should work now in 7.0.1.1 as the 1st driver will start without the next but wait when he reached the distance to his startpoint.

**Comment by JoaoRabit** [comment]
> Fahrer 1 beginnt nicht mit der Feldarbeit bevor nicht Fahrer 2 und 3 durch AD auch aktiviert werden, dieses kann aber nicht passieren weil Fahrer 1 nicht den AD Zielpunkt verlässt, weil er drauf wartet das Fahrer 2 und 3 auch mit dem Konvoi starten.
> 
> <img alt="2022-01-31 06_49_31-Farming Simulator 22" width="1383" src="https://user-images.githubusercontent.com/13314443/151746465-eb4b45e9-87f6-4403-9075-0f49243144c8.png">

Hi! What mod is that on the top left that shows a list of the tractors?

**Comment by mdradek** [comment]
> > <img alt="2022-01-31 06_49_31-Farming Simulator 22" width="1383" src="https://user-images.githubusercontent.com/13314443/151746465-eb4b45e9-87f6-4403-9075-0f49243144c8.png">
> 
> Hi! What mod is that on the top left that shows a list of the tractors?

It's "Vehicle Inspector"
